custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -503,6 +503,30 @@
           }
         }
       },
+      {
+        "column_id": "person_case_id",
```

Comment:
The UCR changes will need to be deployed before these aggregation changes, so that when the aggregation runs the first time, these columns aren't empty. You will also need to write a script to migrate data into this as a naive rebuild may take too long. You should be able to grab the information from the ccs record monthly table. For example:

```sql
UPDATE ccs_record_cases_ucr  ccs_ucr SET
   person_case_id = monthly.person_case_id
FROM ccs_record_monthly_tableau AS monthly
WHERE ccs_ucr.person_case_id IS NULL AND ccs_ucr.doc_id = monthly.doc_id
```

That's the general idea at least, there may be a syntax error there, and you'll want to update all the properties instead of just one.

Reply:
hmmmm. Thanks Emord, in that case will separate this ucr update to other PR.